src/UI/Root/Components/Sidebar/Navigation/NavigationItem.tsx

```diff
@@ -26,8 +27,15 @@ export const NavigationItem: React.FC<Link> = ({
   external,
   locked,
 }) => {
-  if (locked) return <LockedItem label={label} key={id} />;
-  if (external) return <ExternalItem label={label} url={url} key={id} />;
+  if (locked) {
+    return <LockedItem label={label} key={id} />;
+  }
+  if (external) {
+    return <ExternalItem label={label} url={url} key={id} />;
+  }
+  if (id === "CompileReports") {
```

Comment:
The day you change the id, it won't work anymore. I think it would have been better to add an optional property to the Link Interface, just like it's done for the other elements. I think that would be more dynamic and flexible.

Also, maybe something to discuss later, but I don't know if you know the concept of 'magic' strings. This is what I consider a magic string, and usually, that's something you would rather store in a const.